modules/Sources/UIComponents/Chips/WordChipDropDelegate.swift

```diff
@@ -9,7 +9,6 @@ import Foundation
 import SwiftUI
 import OrderedCollections
 import ComposableArchitecture
-import UIComponents
 
 /// Drop delegate that accepts items conforming to `PhraseChip.validationWordTypeIdentifier`
 public struct WordChipDropDelegate: DropDelegate {
```

Comment:
Are you keeping the backup puzzle? 🥹

Reply:
No, the Nighthawk designs don't include it. I haven't taken the time to remove the default secant UI because we have as of today, tried to stay synced with upstream. Again, I think it may be time to break the tie and let Nighthawk 2.0 exist and evolve on its own, apart from the reference wallet from which it is derived. But until we make that decision as an org, stuff like this stays. There was a cyclic dependency I was running into due to a `Model` object importing `UIComponents`